apps/web/src/components/AddressMevLookup.tsx

```diff
@@ -196,6 +198,8 @@ const AddressMevLookup = () => {
             src="/images/image-waddles-1.png"
             alt="darklake waddles 1"
             className="relative lg:bottom-[-40px] hidden lg:block"
+            width={24}
+            height={24}
```

Comment:
wrong sizes, but this is sorted in #279 

```suggestion
            width={24}
            height={24}
```